src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,61 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
```

Comment:
It is recommended to use a variable for the number of columns in the grid. This will make it easier to adjust the grid layout in the future. Example: $columns: 12;